<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1476</width>
    <height>861</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QPushButton" name="managerSelectButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manager</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientSelectButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>0</y>
     <width>20</width>
     <height>761</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-20</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>321</width>
     <height>81</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Please choose your Role :</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>650</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You selected : None</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputJahr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputPreis">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputModell">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputMarke">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>User Input Area</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modell :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Marke :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Jahr :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Preis :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Treibstoff :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputTreibstoff">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>650</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputKilometer">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Kilometer :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputLeistung">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>590</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Leistung :</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>570</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>630</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>861</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>20</width>
     <height>701</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>User Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fugeEin">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Einfugen eines Autos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="losche">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Loschen eines Autos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aktualisiere">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktualisieren eines Autos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sucheModell">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen eines Modells</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sucheMarke">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen einer Marke</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filterAlter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtern nach Alter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filterKilometer">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtern nach Kilometer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeigeNachPreisSortiert">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeigen sortiert nach Preis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fugeZurFavoritenEin">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Einfugen zur Lieblingsautos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeigeFavoriten">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>600</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzeigen der Lieblingsautos</string>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>750</y>
     <width>861</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="operationStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>720</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>- Last Operation status -</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="username">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Username&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
